core/src/test/java/org/elasticsearch/common/unit/FuzzinessTests.java

```diff
@@ -90,23 +91,26 @@ public void testParseFromXContent() throws IOException {
             }
             {
                 XContentBuilder json;
-                if (randomBoolean()) {
+                boolean defaultAutoFuzziness = randomBoolean();
+                if (defaultAutoFuzziness) {
                     json = Fuzziness.AUTO.toXContent(jsonBuilder().startObject(), null).endObject();
                 } else {
                     String auto = randomBoolean() ? "AUTO" : "auto";
-                    if (randomBoolean()){
-                        auto += ":" + randomIntBetween(0, 3) + "," + randomIntBetween(3, 10);
+                    if (randomBoolean()) {
+                        auto += ":" + randomIntBetween(1, 3) + "," + randomIntBetween(4, 10);
                     }
                     json = jsonBuilder().startObject()
-                            .field(Fuzziness.X_FIELD_NAME, auto)
-                            .endObject();
+                        .field(Fuzziness.X_FIELD_NAME, auto)
+                        .endObject();
                 }
                 XContentParser parser = createParser(json);
                 assertThat(parser.nextToken(), equalTo(XContentParser.Token.START_OBJECT));
                 assertThat(parser.nextToken(), equalTo(XContentParser.Token.FIELD_NAME));
                 assertThat(parser.nextToken(), equalTo(XContentParser.Token.VALUE_STRING));
                 Fuzziness parse = Fuzziness.parse(parser);
```

Comment:
I think that you should think of a name that is more descriptive for the variable "parse"